tests/integration/cassandra/certs/README.md

```diff
@@ -0,0 +1 @@
+password=cassandra
```

Comment:
can we add few lines regarding the tests and ho to leverage the password?

Reply:
maybe it would also add few lines in a new https://github.com/newrelic/nri-cassandra/tree/master/tests/integration/README.md now that you have the context